api/appendix_e.asciidoc

```diff
@@ -85,11 +85,19 @@ The following features are added to the OpenCL C programming language
 The following features in OpenCL 1.0 are deprecated (see glossary) in OpenCL
 1.1:
 
-  * The {clSetCommandQueueProperty} API is no longer supported in OpenCL
-    1.1.
-  * The `__ROUNDING_MODE__` macro is no longer supported in OpenCL C 1.1.
-  * The cl-strict-aliasing option that can be specified in _options_
-    argument to {clBuildProgram} is no longer supported in OpenCL 1.1.
+// Bugzilla 6140
+  * The {clSetCommandQueueProperty} API is deprecated, which simplifies
+    implementations and possibly improves performance by enforcing that
+    command queue properties are invariant.
+    Applications are encouraged to create multiple command queues with
+    different properties versus modifying the properties of a single
+    command queue.
+// Bugzilla 6628
+  * The `-cl-strict-aliasing` build option has been deprecated.
+    It is no longer required after defining type-based aliasing rules.
+// Bugzilla 5593 and 6068
+  * The *cl_khr_select_fprounding_mode* extension is deprecated and its
```

Comment:
No much of rationale is described here :)

Reply:
I've merged this PR, this point was never resolved but I don't know what else could be said.